content/docs/getting-started/faq.md

```diff
@@ -0,0 +1,90 @@
+- **The Tina Paradigm:** Your website is it's own CMS. You use Tina to build content editing capabilities directly into your website interface. You can set up a custom or pre-configured backend to handle how and where content is stored.
+
+Tina is not an application on its own that you login to, however you could build that authenticated editing application into your site with Tina. It is not a one-size-fits all solution. It is incredibly flexible, providing precise control over an entire content editing infrastructure — from the database to the user experience.
+
+## Is Tina stable?
+
+Currently, Tina is pre Version-1. While much of the core architecture is fairly-well settled, many of the higher-level APIs are still being built and shaped. Breaking changes in non-core packages are introduced regularly (but communicated in Release Notes) as we push limits and hone the developer experience.
+
+## What framework do you best support?
+
+Tina's editing UI is built entirely in React. Right now, you can only use Tina in React-based environments, such as [CRA](https://reactjs.org/docs/create-a-new-react-app.html), [Next.js](https://nextjs.org/), or [Gatsby](https://www.gatsbyjs.com/).
+
+As of now, we'd recommend using CRA or Next.js if you're wanting to use [Tina in production](/docs/getting-started/faq#what-is-the-editing-environment-how-does-tina-work-in-production) due to a few constraints in the way Gatsby works.
+
+## What about Vue?
+
+While it is entirely possible to use Tina with Vue, currently there are no adaptors, packages, or documented methods of doing this. This is an unbeaten path.
+
+We can identify a few different ways to approach Vue support:
+
+- One way would be to create an adaptor that allows you to set up the React editing UI within your Vue site. This isn't ideal for many reasons, but it is an option.
+- Another would be to completely rewrite the editing UI with Vue components.
+- A third option would be to convert the editing UI to [Web Components](https://developer.mozilla.org/en-US/docs/Web/Web_Components), so they can be independent of a UI framework.
+
+The Core Tina Team is focused on creating a solid foundation with React first, before considering Vue. However, if someone in the community wants to spearhead this effort, we will certainly offer support and guidance.
+
+## Who should use Tina?
+
+From a **development perspective**, folks who are comfortable with React and Javascript.
+
+From a **content editing perspective**, the UX has been created with developers-first in mind. For example, the [GitHub editing workflow](https://tinacms.org/guides/nextjs/github/initial-setup) and interface assumes the content editor is comfortable with Git terminology such as forking, branching, pull requests etc. We chose to focus on developers editing sites before adding abstractions to smooth out the experience for non-technical editors.
```

Comment:
```suggestion
From a **content editing perspective**, the current UX has been created with developers-first in mind. For example, the [GitHub editing workflow](https://tinacms.org/guides/nextjs/github/initial-setup) and interface assumes the content editor is comfortable with Git terminology such as forking, branching, pull requests etc. We chose to focus on developers editing sites before adding abstractions to smooth out the experience for non-technical editors. This strategy has allowed us to quickly expand Tina's core capabilities and improve it's stability.
```

Reply:
Added the last sentence and the word "current" in "the current UX has been crafted.

My main concern is emphasizing that this is not the final UX and that Tina is intended to be for everyone, but we need to take baby steps to get there.